CSCapstone/urls.py

```diff
@@ -10,10 +10,11 @@
 
 urlpatterns = [
 	url(r'^admin/', admin.site.urls),
+	url(r"^search/", include("watson.urls", namespace="watson")),	
 	url(r'^', include('AuthenticationApp.urls')),
+	url(r'^', include('UniversitiesApp.urls')),
 	url(r'^', include('CSCapstoneApp.urls')),
 	url(r'^projects/', include('ProjectsApp.urls')),
     url(r'^', include('CompaniesApp.urls')),
     url(r'^', include('GroupsApp.urls')),
-	url(r'^university/', include('UniversitiesApp.urls')),
```

Comment:
As the Course module is within the UniversitiesApp, I let all matching happen within UniversitiesApp.urls
The alternative would be accessing courses using a url like
/university/courses/.......
The benefit of the /university/courses/....... url is we could have something like
/university/university_slug/courses/sourse_slug

Lmk which alternative sounds better.